napytau/gui/gui_app.py

```diff
@@ -0,0 +1,478 @@
+from typing import List, Tuple
+
```

Comment:
There aren't any Tests for this, please add some. If you aren't sure how or what to test, we can try to come up with some guidelines.